import ExtendableError from '../exception';

console.log("hello")

throw new Error('')
